aspnetcore/razor-components/index.md

```diff
@@ -1,65 +1,34 @@
 ---
 title: Introduction to Razor Components
 author: guardrex
-description: Explore Blazor, a .NET web framework using C#/Razor and HTML that runs in the browser with WebAssembly.
+description: Explore ASP.NET Core Razor Components, a .NET web framework using C#/Razor and HTML.
 monikerRange: '>= aspnetcore-3.0'
 ms.author: riande
 ms.custom: mvc
-ms.date: 01/29/2019
+ms.date: 02/06/2019
 uid: razor-components/index
 ---
 # Introduction to Razor Components
 
-By [Steve Sanderson](http://blog.stevensanderson.com), [Daniel Roth](https://github.com/danroth27), and [Luke Latham](https://github.com/guardrex)
+By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.com/guardrex)
 
 [!INCLUDE[](~/includes/razor-components-preview-notice.md)]
 
-*ASP.NET Core Razor Components* executes server-side in ASP.NET Core, while *Blazor* (Razor Components executed client-side) is an experimental .NET web framework using C#/Razor and HTML that runs in the browser with WebAssembly. Blazor provides all of the benefits of a client-side web UI framework using .NET on the client.
+*Razor Components* is a .NET web framework using C#/Razor and HTML that makes web development easier and more productive:
```

Comment:
I'd rather not refer to Razor Components as a framework. Blazor and ASP.NET Core are the frameworks that leverage Razor Components.

How about this instead?:

"Razor Components are a new way to build interactive client-side web UI with .NET that is easier and more productive:"

Reply:
Yes, but *is* over *are*, since RC is treated as a singular noun.

... like "darts" - :dart: *Razor Darts*:tm: is a great game. 🎯 :smile:

I'll try changing "easy" because I think that's another :x: **_banned word_** :x: on this repo.